### DISCUSSION TITLE
Why does useRuntimeConfig require a Nuxt instance to access environment variables?

### DISCUSSION THREAD

**Discussion by rmarquet21** [nuxt/nuxt#27541] [Questions]
I'm wondering why we need an instance of Nuxt to access environment variables using `useRuntimeConfig`. Currently, I'm facing a problem where I'd like to call a function in my code that uses `useRuntimeConfig` and call it in my `seed.ts` prisma, which is outside the Nuxt code.

seed.ts
```ts
import { PrismaClient } from '@prisma/client';
import { hashPassword } from '../server/utils/hashPassword'; // <--- Error because it's outside of Nuxt

const prisma = new PrismaClient();

async function seed() {
  const hashedPassword = hashPassword('password');
  await prisma.users.create({ data: { email: 'user@example.com', password: hashedPassword } });
}

seed()
  .catch((error) => {
    process.exitCode = 1;
  })
  .finally(() => prisma.$disconnect());
```

hashPassword.ts
```ts
const config = useRuntimeConfig(); // <--- This requires a Nuxt instance

export function hashPassword(password: string) {
  const { saltRounds } = config; 
  const hashedPassword = bcrypt.hashSync(password, saltRounds);
  return hashedPassword;
}
```

**Comment by TheAlexLichter** [comment]
Because Nuxt/nitro ensure it will load/provide the values.

You can use process.env in scripts you run on your own, or, even better, the [Nitro Task API](https://github.com/unjs/nitro/issues/1974).

**Reply by rmarquet21** [reply]
I get this error if I want to use the nitro cli on a nuxt project...

`Missing info file: .../.nitro/nitro.json (is dev server running?)`
